libusb/os/darwin_usb.h

```diff
@@ -119,6 +119,7 @@ struct darwin_cached_device {
   int                   refcount;
   bool                  in_reenumerate;
   int                   capture_count;
+  usbi_mutex_t          capture_mutex;
```

Comment:
What is this mutex protecting? `capture_count`? Anything else?

Reply:
Eventually (https://github.com/libusb/libusb/pull/1419/commits/d914239453070ce8941635efc92116861c739c4b) we can tag the variable that is protected by a mutex.  So concretely, what would be good to do now when adding these new mutexes is something like a comment:

```
  usbi_mutex_t          capture_mutex;
  int                   capture_count; // GUARDED_BY(capture_mutex)
```

And in fact it's better for -Wthread-safety to declare the mutex before like in https://github.com/libusb/libusb/pull/1694/commits/2e35caa508d6edd073ff929572edf9ce6b30294a.

Reply:
Correct, the mutex is protecting just capture_count (I was hoping the naming would make this obvious). Do you want me to add the comment? I don't see it on any other lock so would it be better to just have it as part of #1419?

Reply:
Indeed the naming does make it pretty obvious, but still... please add the comment, because now is the time that we know the answer. :)  And please put the mutex before.